Nbuild/Program.cs

```diff
@@ -20,23 +22,28 @@ public class Program
     static int Main(string[] args)
     {
         Colorizer.WriteLine($"[{ConsoleColor.Yellow}!{Nversion.Get()}]\n");
-        var result = ResultHelper.New();
-        string? target = null;
-        Cli options;
 
-        if (args.Length == 0)
+        var result = ResultHelper.New();
+        var ParserOptions = new ParserOptions
         {
-            options = new Cli() { Verbose = true };
-            result = BuildStarter.Build(target, options.Verbose);
-        }
-        else if (args.Length == 1 && !args[0].Contains(CmdHelp, StringComparison.InvariantCultureIgnoreCase))
+            LogParseErrorToConsole = false,
+        };
+        var optionsParsed = Parser.TryParse(args, out Cli? options, ParserOptions);
+        if (!optionsParsed)
         {
-            target = args[0];
-            options = new Cli() { Verbose = true };
-            result = BuildStarter.Build(target, options.Verbose);
+            GitWrapper gitWrapper = new();
```

Comment:
This is the  main purpose of the PR is here,  Execution of NB stops if Git is not configured correctly